external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -46,33 +50,33 @@ A column is assumed to be a dimension unless otherwise configured using the `typ
 }
 ```
 
-**If a column mapping is not defined in the `qube-config.json` file for a given CSV column, the column is configured using [convention](./convention.md).**
+**If a column mapping is not defined in the `qube-config.json` file for a given CSV column, the column is [configured by convention](./convention.md).**
 
 ### Dimensions
 
-> The *dimension* [column] serve[s] to identify the observations. A set of values for all the dimension components is sufficient to identify a single observation. Examples of dimensions include the time to which the observation applies, or a geographic region which the observation covers.
+> The *dimension* [column] serves to identify the observations. A set of values for all the dimension components is sufficient to identify a single observation. Examples of dimensions include the time to which the observation applies, or a geographic region which the observation covers.
 
 Think of the principle of [MECE](https://en.wikipedia.org/wiki/MECE_principle).
 
 ### Measures
 
-> The *measure* [column] represent[s] the phenomenon being observed.
+> The *measure* [column] represents the phenomenon being observed.
 
 The measure column is effectively another form of dimension.
 
 ### Attributes
 
-> The *attribute* [column] allow us to qualify and interpret the observed value(s). They enable specification of the units of measure, any scaling factors and metadata such as the status of the observation (e.g. *estimated*, *provisional*).
+> The *attribute* [column] allows us to qualify and interpret the observed value(s). This enables specification of the units of measure, any scaling factors and metadata such as the status of the observation (e.g. *estimated*, *provisional*).
 
-The attribute column can link to resources or literals.
+The attribute column can link to [resources](../../glossary/index.md#uri) or [literals](../../glossary/index.md#literal).
 
 ### Units
 
-The *unit* component is a type of attribute column which provides the units of the observation.
+The *unit* column is a type of attribute column which provides the units of the observation.
 
 ### Observations
 
-The *observation* column are the numeric values of the observation being recorded in the data set.
+The *observation* column contains the numeric values of the observations recorded in the data set.
 
 ### Using templates
 
```

Comment:
I followed up on your notes about the template page this links to having a broken link for the available templates(https://github.com/GSS-Cogs/csvcubed/tree/main/csvcubed/csvcubed/readers/cubeconfig/v1_0/templates) and you are right, it is incorrect. I believe the correct link is https://github.com/GSS-Cogs/csvcubed/tree/main/src/csvcubed/readers/cubeconfig/v1_0/templates
I know it is not part of the qube config documentation but well done for noticing that anyway

Reply:
Thanks, amended the link